### PR TITLE
(maint) - fix app credentials invalid char

### PR DESCRIPTION
**Pull request by jordanbreen28**

### CODE REVIEW

**Comment by jordanbreen28** [comment]
this has been tested against latest published container https://github.com/puppetlabs/cat-team-github-metrics/actions/runs/6108667202/job/16578121282

File: .github/workflows/metrics.yml [hunk 1]
```diff
@@ -34,13 +34,15 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
     - name: Collect metrics
       run: |
+        echo '${{ secrets.GCP_CONNECTION }}' > $(pwd)/creds.json
         docker run \
         -e BQ_PROJECT_ID=${{ secrets.BQ_PROJECT_ID }} \
         -e GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} \
         -e COBRA_COMMAND=export \
-        -e 'CONNECTION_KEY=${{ secrets.GCP_CONNECTION }}' \
+        -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/creds.json \
```
Comment (GSPatton) [inline .github/workflows/metrics.yml:42]: i'm assuming this path & file exist from one of the packages you install in your Dockerfile change?

Comment (jordanbreen28) [reply .github/workflows/metrics.yml:42]: if you look couple lines below that we create the docker volume, which copies over the file echoed to prior to the docker run command
this is the suggested use as per the readme and also saves the pain of passing in an unformatted service key :)

Comment (GSPatton) [reply .github/workflows/metrics.yml:42]: ah missed that. Perfect

**Review by GSPatton — APPROVED** [review]

**Pull request merged**